app/settings/users/create.controller.js

```diff
@@ -30,44 +30,50 @@ function (
         $rootScope.$emit('setPageTitle', title);
     });
 
+    $scope.isValid = true;
     $scope.passwordShown = true;
     $scope.user = { role: 'user' }; // @todo don't hardcode default role
     $scope.save = $translate.instant('app.save');
     $scope.saving = $translate.instant('app.saving');
     $scope.saving_user = false;
     $scope.saveUser = function (user) {
         $scope.saving_user = true;
-
-        UserEndpoint.saveCache(user).$promise.then(function (response) {
-            if (response.id) {
-                Notify.notify('notify.user.save_success', {name: user.realname});
-                $scope.saving_user = false;
-                $scope.userSavedUser = true;
-                $scope.user.id = response.id;
-                // in favor of $route.reload();
-                $state.go('settings.users', null, { reload: true });
+        if ($scope.form.$valid) {
+            $scope.isValid = true;
+            UserEndpoint.saveCache(user).$promise.then(function (response) {
+                if (response.id) {
+                    Notify.notify('notify.user.save_success', {name: user.realname});
+                    $scope.saving_user = false;
```

Comment:
I think we could do savingUser instead to keep it similar to the others (userSaveUser etc)